proof/crefine/ARM/Detype_C.thy

```diff
@@ -1428,7 +1428,7 @@ lemma deleteObjects_ccorres':
              doMachineOp_modify modify_modify o_def ksPSpace_ksMSu_comm
              bind_assoc modify_machinestate_assert_cnodes_swap
              modify_modify_bind)
-  apply (rule ccorres_stateAssert_fwd)
+  apply (rule ccorres_stateAssert_fwd)+
```

Comment:
not much of a save

Reply:
We now have two asserts in `deleteObject` so the `+` allows us to gobble up all of them easily. 

For MCS, we would often add a bunch of asserts to the beginnings of functions (for `sym_refs (state_refs_of' s)`, `valid_idle` etc. etc.) and a `+` or the `intro` method for some assert rule worked well. It's not really intended to save lines, exactly, but hopefully make the proofs a little more resilient to further asserts being added. But I can change this if that's not a good approach

Reply:
No, I missed it. the combination of adding an assert *and* a plus looks like it doesn't save a line, but it does. All good.